tutor/specs/e2e/add-edit-question.e2e.js

```diff
@@ -1,12 +1,10 @@
-import { visitPage, setTimeouts, setRole } from './helpers'
-
-// the BE mock api server is primarily in backend/exercises and backend/terms
+import { visitPage, setTimeouts, loginAs } from './helpers'
 
 describe('Add/Edit Questions', () => {
 
     beforeEach(async () => {
         await setTimeouts()
-        await setRole('teacher')
+        await loginAs({ page, userName: 'reviewteacher' })
```

Comment:
sorry, I changed the `loginAs` to take the username as a string,  and an optional page.  This should now be: `loginAs('review teacher')`